draft-ietf-ace-oscore-profile.xml

```diff
@@ -642,7 +715,7 @@ OSCORE_Security_Context = {
       <section anchor="tok-ver" title="Access rights verification">
 
         <t>
-          The RS MUST follow the procedures defined in section 5.8.2 of <xref target="I-D.ietf-ace-oauth-authz"/>: if an RS receives an OSCORE-protected request from a client, then it processes according to <xref target="I-D.ietf-core-object-security"/>. If OSCORE verification succeeds, and the target resource
+          The RS MUST follow the procedures defined in section 5.8.2 of <xref target="I-D.ietf-ace-oauth-authz"/>: if an RS receives an OSCORE-protected request from a client, then it processes it according to <xref target="I-D.ietf-core-object-security"/>. If OSCORE verification succeeds, and the target resource
```

Comment:
Don't use it twice close together.  then the OSCORE request processes the RS? (yes I know that is wrong).)

Reply:
Ok, good point. I will try to rephrase.

Reply:
Rephrased in cd0d68e